### PR TITLE
Add missing levels to sort

### PR DESCRIPTION
**Pull request by brettski**

### CODE REVIEW

File: src/dataSources/cloudFirestore/partner.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ function partnerCollection(dbInstance) {
       ...d.data(),
     }));
 
-    const sortBy = ['PIONEER', 'EXPLORER', 'SCOUT', 'MEDIA'];
+    const sortBy = ['PIONEER', 'EXPLORER', 'SCOUT', 'CUB', 'PATRION', 'MEDIA'];
```
Comment (csell5) [inline src/dataSources/cloudFirestore/partner.js]: shouldn't it be `PATRON`?

Comment (brettski) [reply src/dataSources/cloudFirestore/partner.js]: LOL

Comment (csell5) [reply src/dataSources/cloudFirestore/partner.js]: actually we should also add last years levels

Comment (csell5) [reply src/dataSources/cloudFirestore/partner.js]: just add these to the end of that array and it would work out.  PLATINUM, GOLD, SILVER, BRONZE

Comment (brettski) [reply src/dataSources/cloudFirestore/partner.js]: Resolved last push

Comment (brettski) [reply src/dataSources/cloudFirestore/partner.js]: @csell5 at end, or before MEDIA?

Comment (brettski) [reply src/dataSources/cloudFirestore/partner.js]: Previous levels added

**Pull request merged**